recipes/crbeam/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "crbeam" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/okolo/mcray/archive/refs/tags/v1.1.1rc1.tar.gz
```

Comment:
This indicates a different version than the package version. And we should not ship release candidates I think

Reply:
Ideally you can reuse here the template variable 'version' to construct the URL. This will also enable our bot to create release updates for the package

Reply:
Thank you @bgruening ! I made the changes suggested.